docs(contributing): rotate SECRET_KEY without invalid decryption key error

pankajsoni22 opened the pull request:
docs: rotate SECRET_KEY without invalid decryption key error


### SUMMARY

### If someone has superset running with important stuffs and wants to change SECRET_KEY then definitely they need PREVIOUS_SECRET_KEY mentioned along with SECRET_KEY so decryption with PREVIOUS_SECRET_KEY can happen and encryption with latest SECRET_KEY can be performed. But in the documentation a wrong default SECRET_KEY is mentioned that can mislead the reader/user.

pankajsoni22 committed:
Update default SECRET_KEY in docs

pankajsoni22 commented:
@rusackas How can I contribute to docs.

nytai commented on docs/docs/installation/configuring-superset.mdx:
````diff
@@ -285,7 +285,7 @@ If you want to rotate the SECRET_KEY(change the existing secret key), follow the
 ```python
-PREVIOUS_SECRET_KEY = 'CURRENT_SECRET_KEY' # The default SECRET_KEY for deployment is '21thisismyscretkey12eyyh'
-SECRET_KEY = 'YOUR_OWN_RANDOM_GENERATED_SECRET_KEY'
+PREVIOUS_SECRET_KEY = 'CURRENT_SECRET_KEY' # The default SECRET_KEY for deployment is mentioned in superset/constants.py as CHANGE_ME_SECRET_KEY variable
````
The default secret key has changed a few times. Might be worth mentioning how they can find it via their previous deployment. Something like 

```shell
$ superset shell
>>> from flask import current_app; print(current_app.config["SECRET_KEY"])
```

pankajsoni22 replied:
> The default secret key has changed a few times. Might be worth mentioning how they can find it via their previous deployment. Something like
> 
> ```shell
> $ superset shell
> >>> from flask import current_app; print(current_app.config["SECRET_KEY"])
> ```

I think that's a good Idea. Thanks for your suggestion @nytai

pankajsoni22 committed:
docs(contributing): create or change secret key

Creating and changing secret key may be challenging if we don't know what is the current secret key so find out in a better way....

pankajsoni22 commented:
Creating one more pull request on top of this.
